website/assets/styles/layout.less

```diff
@@ -213,6 +213,10 @@ body.detected-mobile {
   // …
 }
 
+#dropdownMenuButton {
+  cursor: unset;
+}
+
```

Comment:
- Let's stick to kebab case rather than camelCase
- Instead of an id, use `[purpose='dropdown-menu-button'] {}`  (I know the id was already there- we can change it to `purpose="dropdown-menu-button"` in the HTML instead, and make the selector `[purpose='dropdown-menu-button'] {}`)